boards/infineon/cy8cproto_062_4343w/doc/index.rst

```diff
@@ -101,50 +102,58 @@ To fetch Binary Blobs:
 Build blinking led sample
 *************************
 
-Here is an example for the :zephyr:code-sample:`blinky` application.
+Here is an example for building the :zephyr:code-sample:`blinky` sample application.
 
-.. code-block:: console
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: cy8cproto_062_4343w
+   :goals: build
 
-   cd zephyr
-   west build -p auto -b cy8cproto_062_4343w samples/basic/blink
+Programming and Debugging
+*************************
 
-OpenOCD Installation
-====================
+The CY8CPROTO-062-4343W includes an onboard programmer/debugger (`KitProg3`_) to provide debugging, flash programming, and serial communication over USB. Flash and debug commands use OpenOCD and require a custom Infineon OpenOCD version, that supports KitProg3, to be installed.
```

Comment:
Previously these lines were wrapped at 80 columns now they seem a little longer.  Are you sure that is what you want?  Commenting on this only once as it spans a few different diff sections.